index.js

```diff
@@ -0,0 +1,50 @@
+(function(global, f){
+
+  'use strict';
+
+  /*istanbul ignore next*/
+  if(module && typeof module.exports !== 'undefined'){
+    module.exports = f(
+      require('fluture'),
+      require('sanctuary-def'),
+      require('sanctuary-type-identifiers')
+    );
+  }else{
+    global.concurrify = f(
```

Comment:
A copy-paste oversight, as I copied the boilerplate from `concurrify`. :neutral_face: